src/node/http.ts

```diff
@@ -69,6 +71,29 @@ export const authenticated = (req: express.Request): boolean => {
           ? safeCompare(req.cookies.key, req.args["hashed-password"])
           : req.args.password && safeCompare(req.cookies.key, hash(req.args.password)))
       )
+    case AuthType.Openid:
+      console.log(req.oidc.user)
```

Comment:
It looks like we might want to use `console.debug` and add some other text here? It's definitely helpful to have debug output as figuring out OIDC integration problems can be... a challenge :sweat_smile: but we'd some other text that we can search for, so we know where the logs are coming from.

maybe something like:

```javascript
console.debug("authenticated using OpenID Connect as", req.oidc.user)
```

other debug info like the returned OIDC claims seems useful too? they're secret credentials, but since code-server is meant for a single user, maybe logging those is OK?

Reply:
Yes, very sorry, I'll include this change along with the updates to yarn lockfile.

Reply:
No apologies needed! Thanks so much for your contribution! :heart: